bin/db-initialization.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
+import os
+
+from buildpg import V, asyncpg, render
```

Comment:
You aren't using the special `buildpg` extensions to `asyncpg`, so it seems like this should be a vanilla `asyncpg` import.